replica/transition_test.go

```diff
@@ -1 +1,298 @@
 package replica_test
+
+import (
+	"testing/quick"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/renproject/hyperdrive/block"
+	. "github.com/renproject/hyperdrive/replica"
+)
+
+var conf = quick.Config{
+	MaxCount:      256,
+	MaxCountScale: 0,
+	Rand:          nil,
+	Values:        nil,
+}
+
+var _ = Describe("replica TransitionBuffer", func() {
+	Context("When only given Proposed Transitions", func() {
+		It("Number of Enqueue matches Dequeue for same height", func() {
```

Comment:
An `It` statement should also begin with `"should.."`. For example:  `It("should enqueue the same number of times as it dequeues for a given height", func() { .. })`. Don't be afraid to embed contexts inside contexts inside contexts to add more `"when..."`.

Reply:
I did not know (or realize) I could embed context inside of context.